### PR TITLE
Added VPN Tunnel Phase 1/2 Lifetimes

### PR DESCRIPTION
**Pull request by jpschafer**
## what

Added Tunnel 1/2 Phase 1/2 Lifetime Variables

## why

So I can add custom lifetimes outside of the defaults for customers who have different lifetimes set for their end of the tunnel.

## references

#54 

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpn_connection#tunnel1_phase1_lifetime_seconds

The only issue to note is that in the AWS Console,  you will see 28800 and 3600 respectively instead of "default" for each. I wasn't sure If that really matters but it's the only issue I could think of. I'm open to suggestions on preferences regarding this variable that is technically optional. I wasn't sure if setting it to null would create the same behavior.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -87,7 +89,9 @@ resource "aws_vpn_connection" "default" {
   tunnel2_phase2_encryption_algorithms = var.vpn_connection_tunnel2_phase2_encryption_algorithms
   tunnel2_phase1_integrity_algorithms  = var.vpn_connection_tunnel2_phase1_integrity_algorithms
   tunnel2_phase2_integrity_algorithms  = var.vpn_connection_tunnel2_phase2_integrity_algorithms
-
+  tunnel2_phase1_lifetime_seconds      = var.vpn_connection_tunnel1_phase1_lifetime_seconds 
+  tunnel2_phase2_lifetime_seconds      = var.vpn_connection_tunnel1_phase2_lifetime_seconds
```
Comment (gberenice) [inline main.tf:93]:
```suggestion
  tunnel2_phase1_lifetime_seconds      = var.vpn_connection_tunnel2_phase1_lifetime_seconds 
  tunnel2_phase2_lifetime_seconds      = var.vpn_connection_tunnel2_phase2_lifetime_seconds 
```

**Review by gberenice — CHANGES_REQUESTED** [review]
@jpschafer could you please fix the variable usage as suggested and run `fmt`?

File: main.tf (hunk 1 above)
Comment (jpschafer) [reply main.tf]: Sorry about that, It's corrected.

**Review by gberenice — APPROVED** [review]
Thanks @jpschafer!

**Comment by jpschafer** [comment]
@gberenice Just wanted to say thank you for helping me out in merging this and letting me contribute to this project. You all do great work! :)

**Pull request merged**